Lib/test/test_dbm.py

```diff
@@ -155,6 +155,21 @@ def test_keys(self):
             self.assertNotIn(b'xxx', d)
             self.assertRaises(KeyError, lambda: d[b'xxx'])
 
+    def test_clear(self):
+        with dbm.open(_fname, 'c') as d:
+            self.assertEqual(d.keys(), [])
+            a = [(b'a', b'b'), (b'12345678910', b'019237410982340912840198242')]
```

Comment:
The second entry is a little bit complicated but it's just a remark, feel free to ignore 😁

Reply:
Yeah it was just copied value from L148 :)